server/src/projects/projects.attrs.ts

```diff
@@ -5,6 +5,8 @@ export const PROJECT_ATTRS = [
   'statecode',
   'statuscode',
   'dcp_visibility',
+  'dcp_publicstatus',
```

Comment:
Adding this in another PR #527 but don't think it should cause much conflict!

Reply:
yep, frontend and backend! :)